Makefile

```diff
@@ -12,13 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION=v0.9.0
+VERSION?=v0.9.0
```

Comment:
nit: Is there more context behind this change? Is there any world in which VERSION would be already set before making this declaration?